projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -38,7 +37,7 @@
   void disablePlaceholder();
   void setItemFactory(Supplier<SourceT> itemFactory);
   void setSeparator(Character ch);
-  void replaceWithNewOnRemove(Predicate<SourceT> shouldReplace);
+  void setReplaceNonemptyWithNewOnRemove(boolean replaceWithNew);
```

Comment:
Can we remove this method? I think we always want this (replaceWithNew == true).